code/package.json

```diff
@@ -79,7 +80,10 @@
     "eslint-plugin-truffle": "^0.3.1",
     "moment-timezone": "^0.5.33",
     "rlp": "^2.2.6",
+    "solc-0.8.4": "npm:solc@^0.8.4",
     "solhint": "^3.3.5",
+    "solidity-coverage": "^0.7.20",
```

Comment:
some dependencies including this should be devDependencies

Reply:
moved